plugins/CoreHome/vue/src/ReportingPage/ReportingPage.vue

```diff
@@ -0,0 +1,292 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <div class="reporting-page">
+    <ActivityIndicator
+      :loading="loading"
+    />
+    <div v-show="hasNoPage">{{ translate('CoreHome_NoSuchPage') }}</div>
+    <div
+      class="row"
+      v-for="widget in widgets"
+      :key="widget.uniqueId"
+    >
+      <Widget
```

Comment:
I guess we need to add some special class or attribute here. We have some javascript looking for `[piwik-widget]`, which no longer does exist. See https://github.com/matomo-org/matomo/blob/b7bfb36a3d37e4d056ab3f0355672b85f318f3d9/plugins/CoreHome/javascripts/dataTable.js#L1934

This also causes the UI test failure where a tooltip is still visible.

Reply:
:+1: nice catch, I'll put looking for selectors like this on my checklist.